docs/azure/sdk/snippets/aspnetcore-guidance/BlazorSample/BlazorSample.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings>enable</ImplicitUsings>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Azure.Messaging.ServiceBus" Version="7.18.2" />
+    <PackageReference Include="Azure.Security.KeyVault.Secrets" Version="4.7.0" />
+    <PackageReference Include="Azure.Storage.Blobs" Version="12.22.2" />
+    <PackageReference Include="Microsoft.Extensions.Azure" Version="1.7.6" />
```

Comment:
I think there will be a new 1.8.0 going out this release cycle.  Not sure if we would want to update this when that happens?

Reply:
It's available now: https://www.nuget.org/packages/Microsoft.Extensions.Azure/1.8.0. I agree we should update the dependency.